R-packages/covidcast/R/cor.R

```diff
@@ -1,13 +1,18 @@
 #' Compute correlations between two `covidcast_signal` data frames
 #'
-#' Computes correlations between two `covidcast_signal()` data frames, allowing
-#' for slicing by geo location, or by time. (The latest issue from each data
-#' frame is used when computing correlations.) See
-#' `vignette("correlation-utils")` for examples.
+#' Computes correlations between two `covidcast_signal` data frames, allowing
+#' for slicing by geo location, or by time. (Only the latest issue from each
+#' data frame are used for correlations.) See `vignette("correlation-utils")`
+#' for examples.
 #'
 #' @param x,y The `covidcast_signal` data frames to correlate.
 #' @param dt_x,dt_y Time shifts to consider for `x` and `y`, respectively,
-#'   before computing correlations. These must both be nonnegative, and only one
+#'   before computing correlations. Negative shifts translate into in a lag
+#'   value and positive shifts into a lead value; for example, if `dt = -1`,
+#'   then the value on June 2 that gets reported is the original value on June
+#'   1; if `dt = 0`, then the values are left as is.
+#'
+#' These must both be nonnegative, and only one
 #'   can be positive. If `dt_x = 1`, for example, then data for `x` is shifted
 #'   forward 1 day in time (so, data on June 1 becomes data on June 2, and so
 #'   on). Default is 0 for both.
```

Comment:
Is this documentation accurate anymore? It still says both must be nonnegative.

Reply:
Oops that was a paragraph that was intended to be deleted. Good catch, thanks.